numpy/core/src/common/simd/intdiv.h

```diff
@@ -0,0 +1,485 @@
+/**
+ * This header implement intrinsics npyv_divisor_* that used
+ * for computing the parameters of fast integer division.
+ */
+#ifndef NPY_SIMD
+    #error "Not a standalone header, use simd/simd.h instead"
+#endif
+#ifndef _NPY_SIMD_INTDIV_H
+#define _NPY_SIMD_INTDIV_H
+/*****************************************************************************
+ ** Integer division operators
+******************************************************************************
+*
+* NOTES:
+*  - The fast integer division adopted from Agner Fog's work,
+*    the original code is part of the VCL library:
+*    https://github.com/vectorclass/version2/blob/46a58414bc6128b61d77186656860ca7b0b961d8/vectori128.h#L6239
```

Comment:
I don't understand. Either the code is a derivative of the VCL library by Agner Fog, in which case we need to credit them and follow their license, or it is not a derivative (it is a clean room reimplementation from the original paper), and we do not need to mention their implementation.

Reply:
I made a second cleanup to remove any comments and code related to VCL, the final code is now 100% follows the original paper.